src/main/java/robot/subsystems/gripper/GripperSubsystem.java

```diff
@@ -0,0 +1,65 @@
+package robot.subsystems.gripper;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import robot.RobotMap;
+
+public class GripperSubsystem extends Subsystem {

```

Comment:
```suggestion
public class Gripper extends Subsystem {
```
The word `subsystem` is redundant. Remove it using IntelliJ IDEA's refactoring options.

Reply:
I disagree, I think we need to keep the word "Subsystem" because it's helping to understand the purpose of this File before you start to diff inside him, I meant you will understand what this class intent to be.

Reply:
There really is no chance you could think "Gripper" means anything other than the name of a subsystem. Also, this line specifically says `extends Subsystem` so it's crystal clear `Gripper` is a subsystem.

Reply:
I meant in the name of the file not in the file's content

Reply:
I talked about the same thing. I talked about the class name (which is identical to the file name).

Reply:
We are going to change the structure soon, there is no reason to refactor it to `Gripper` and then refactor everything again after the merge